server.js

```diff
@@ -62,12 +88,13 @@ module.exports= function(onConsoleLog,
     'exception': onException
   };
 
-  var process = function(request, flungLogs) {
+  var processLog = function(request, flungLogs) {
```

Comment:
Yeah for some reason when I ran locally it was treating process like a special thing and erroring just changing the name seems to fix it.